packages/nodejs/src/config.ts

```diff
@@ -36,11 +36,9 @@ export class Configuration {
 
   /**
    * Returns `true` if the current configuration is valid.
-   *
-   * @todo
    */
   public get isValid(): boolean {
-    return true
+    return this.data.apiKey ? true : false
```

Comment:
```suggestion
    return (this.data.apiKey || "").trim() !== ""
```

This was more the implementation I'm looking for. Checking if the Push API key is not an empty string, or string with only spaces.